faunadb-scala/src/main/scala/faunadb/query/package.scala

```diff
@@ -1021,6 +1033,14 @@ package object query {
   def NewId(): Expr =
     Expr(ObjectV("new_id" -> NullV))
 
+  /**
+    * An Access Provider expression.
+    *
+    * '''Reference''': [[https://docs.fauna.com/fauna/current/api/fql/functions/accessprovider]]
+    */
+  def AccessProvider(name: Expr): Expr =
```

Comment:
you miss the version that receives the scope, ie https://github.com/fauna/faunadb-jvm/blob/master/faunadb-scala/src/main/scala/faunadb/query/package.scala#L1055